src/components/tutorial-meta/components/badges/helpers.ts

```diff
@@ -0,0 +1,71 @@
+import React, { ForwardRefExoticComponent, RefAttributes } from 'react'
+import { IconProps } from '@hashicorp/flight-icons/svg-react/types'
+import { IconClock16 } from '@hashicorp/flight-icons/svg-react/clock-16'
+import { IconPlay16 } from '@hashicorp/flight-icons/svg-react/play-16'
+import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal-screen-16'
+import { EditionOption } from 'lib/learn-client/types'
+import { ProductUsed as ClientProductUsed } from 'lib/learn-client/types'
+import {
+  getBadgeComponent,
+  ProductDisplayOption,
+  BadgeComponentProps,
+} from './components/badge'
+import { BadgeOptions } from '.'
+
+type DisplayOptions = {
+  [K in keyof BadgeOptions]: DisplayOption | ProductDisplayOption[]
+}
+
+type DisplayOption = {
+  label: string
+  icon?: ForwardRefExoticComponent<IconProps & RefAttributes<SVGSVGElement>>
+}
+
+const editionDisplayOptions: { [K in EditionOption]: string } = {
```

Comment:
copying these over even though we don't have all the terraform editions in devdot yet.